<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UICompareDock</class>
 <widget class="QDockWidget" name="UICompareDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="doCompareButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>85</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始对比</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tb_infoFileA">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>14</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_infoFileA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="tb_infoFileB">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>44</y>
      <width>37</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_infoFileB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="doClearButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>115</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除对比</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doChangeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>交换</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
